modules/openapi-generator/src/main/resources/kotlin-client/libraries/jvm/infrastructure/ApiClient.kt.mustache

```diff
@@ -21,6 +21,7 @@ import okhttp3.ResponseBody
 import okhttp3.MediaType.Companion.toMediaTypeOrNull
 {{/jvm-okhttp4}}
 import okhttp3.Request
+import okhttp3.Credentials
```

Comment:
@yoshimotocamobile Thanks for your contribution 🙂 
Since https://github.com/OpenAPITools/openapi-generator/pull/4772 was merged and already fixes this, could you please revert this change? Thanks